xdebug.ini

```diff
@@ -1,7 +1,9 @@
-xdebug.mode = debug
-xdebug.client_host = host.docker.internal
-xdebug.idekey = vscode-xdebug
-xdebug.client_port = 9003
-xdebug.start_with_request = trigger
-xdebug.max_nesting_level = 256
-;xdebug.log = /tmp/xdebug.log
\ No newline at end of file
+variables_order = "EGPCS"
+xdebug.mode = debug
+xdebug.client_host = host.docker.internal
+xdebug.idekey = vscode-xdebug
+xdebug.client_port = 9003
+xdebug.start_with_request = trigger
+xdebug.max_nesting_level = 256
+xdebug.output_dir = /app
+xdebug.log = /tmp/xdebug.log
```

Comment:
What does this output directory setting do? And why are you uncommenting the logging line? It's been a while since I've used that logging, but I seem to remember not wanting to leave it enabled for some reason.

Reply:
The [output_dir](https://xdebug.org/docs/profiler#output_dir) is used when xdebug is in profiling mode, it outputs the profiling data to that folder. Defaults to `/tmp` but setting it to `/app` saves it to the host system at the root of repo so you don't have to manually copy it out of the appserver container in order to view the results. 

I can't remember exactly why I enabled the logfile but probably because I was having some issues connecting to xdebug a while back.  I can probably remove that line.